build.sh

```diff
@@ -22,12 +22,77 @@ do
         "-c" | "clean"   ) clean=1;;
         "-C" | "CLEAN"   ) clean=1 && nobuild=1;;
         "-i" | "install" ) install=1;;
+        "-r" | "restart" ) restart=1;;
+        "-f" | "tail"    ) tail=1;;
+        "-d" | "debug"   ) debug=1;;
+        "-D" | "dev"     ) dev=1;;
         "-t" | "test"    ) test=1;;
+        "clean-logs"     ) clean_logs=1;;
         "clang"          ) clang=1;;
         "ci"             ) builddir="cmake_ci/";;
+        "-h"  | "--help" ) help=1;;
     esac
 done
 
+# Display help text
+if [[ -n "${help}" ]]; then
+    cat << EOF
+Usage: $0 [options]
+Helper script simplifying the build process of Pi-hole FTL.
+
+Options:
+  -d, dev            Build, install, restart, and tail logs.
+  -c, clean          Clean the build environment and trigger a build.
+  -C, CLEAN          Clean the build environment and DO NOT build.
```

Comment:
I think it's not very user friendly to have the same option once in lower-case and once in upper-case.